plugins/evernote.koplugin/main.lua

```diff
@@ -139,9 +145,32 @@ function EvernoteExporter:addToMainMenu(tab_item_table)
                 checked_func = function() return self.html_export end,
                 callback = function()
                     self.html_export = not self.html_export
+                    if self.html_export then self.txt_export = false end
+                    self:onSaveSettings()
+                end
+            },
+            {
+                text = _("Export to local clipping text file"),
+                checked_func = function() return self.txt_export end,
+                callback = function()
+                    self.txt_export = not self.txt_export
+                    if self.txt_export then self.html_export = false end
                     self:onSaveSettings()
                 end
             },
+            {
+                text = _("Purge history records"),
+                callback = function()
+                    self.config:purge()
+                    UIManager:show(InfoMessage:new{
+                        text = _("History records are purged.\nAll notes will be exported again next time.\nSuggest to remove existing KOReaderClipping.txt to avoid a duplication."),
+                    })
+                end
+            },
+            {
+                text = _("  - All notes will be exported again next time"),
```

Comment:
Thanks! I think reader should be fine. As I mentioned earlier, kpvbooklet users will probably rely on this behavior.